src/main/java/TaskManager.java

```diff
@@ -0,0 +1,107 @@
+public class TaskManager {
+    protected Task[] taskList;
+    protected int taskCount;
+    TaskManager(int taskCount){
+        this.taskCount = 0;
+        taskList = new Task[100];
+    }
+    public void addTask(Task upcomingTask){
+        taskList[taskCount] = upcomingTask;
+        taskCount += 1;
+    }
+
+    public void markTaskAsDone(int taskIndex){
+        taskList[taskIndex-1].markAsDone();
+    }
+    public void markTaskAsUndone(int taskIndex){
+        taskList[taskIndex-1].markAsUndone();
+    }
+    public void markTask(String input) {
+        int whitespaceIndex = input.indexOf(" ");
```

Comment:
Preferably name whitespaceIndex as whiteSpaceIndex with a capital S (camelCase).

Reply:
Hey khooyourun, thanks for the comment. However, whitespace is a single word